x-pack/test/api_integration/apis/telemetry/index.js

```diff
@@ -8,5 +8,6 @@ export default function ({ loadTestFile }) {
   describe('Telemetry', () => {
     loadTestFile(require.resolve('./telemetry'));
     loadTestFile(require.resolve('./telemetry_local'));
+    loadTestFile(require.resolve('./opt_in'));
```

Comment:
This may be a historical question for someone else, but do you know why these API integration tests are under xpack when the code is in OSS?

Reply:
Hahaha, great question, don't know exactly.  I believe telemetry moved from x-pack to OSS (or at least not x-pack), the code was migrated, and of course it turns out the functional tests didn't actually HAVE to move, and so ... didn't.

Seemed out-of-scope for this PR, but I'll open a new issue to track it.

Reply:
opened issue https://github.com/elastic/kibana/issues/49933 to track this